### PR TITLE
Release note 1.25.0

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit is a release note for 1.25.0.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #11484

### CODE REVIEW

**Comment by hseok-oh** [comment]
It is not included training feature because training feature is not build on release package.

File: docs/release/1.25/release-note_1.25.0.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+## ONE Runtime
+
+- Support ubuntu 20.04
+
+### CPU Backend Operation
+- CPU backend supports per-channel hybrid quantization of int8 type weight and float activation. (TFLite's dynamic range quantization)
+
+### On-device Quantization
+- _onert_ supports new experimental API for on-device quantization.
+- As a 1st step, _onert_ supports per-channel hybrid quantization of int8/int16 type weight and float activation.
```
Comment (glistening) [inline docs/release/1.25/release-note_1.25.0.md:12]:
```suggestion
- As the 1st step, _onert_ supports per-channel hybrid quantization of int8/int16 type weight and float activation.
```

**Review by glistening — DISMISSED** [review]
LGTM (I know it is draft PR).

**Review by ragmani — APPROVED** [review]
LGTM 👍

File: docs/release/1.25/release-note_1.25.0.md (hunk 1 above)
Comment (hseok-oh) [reply docs/release/1.25/release-note_1.25.0.md:12]: :+1:

Comment (glistening) [reply docs/release/1.25/release-note_1.25.0.md]: @hseok-oh As I said offline, I found "as a 1st step" is not incorrect. It is just an idiom. Now, I am okay for both "a" and "the" in the sentence above :)

**Comment by hseok-oh** [comment]
@glistening @ragmani Please approve this.

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**